scripts/kubernetes/sth-service.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: sth-api
+  labels:
+    service: sth-external
+spec:
+  type: NodePort
+  ports:
+  - port: 8000
+    nodePort: 30000
```

Comment:
I suggest to remove that and let k8s to obtain a free port. Also add to readme how to check where sth is exposed.

Reply:
Good one, fixed :)